### PR TITLE
New indicators layouts

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
All the new layouts for indicators according to the stix base reconstruction.

This is the layouts part of - https://github.com/demisto/content/pull/17794

## Screenshots
SCO
image

SDO
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonTypes/ReleaseNotes/3_2_42.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+#### Layouts
+##### Email Indicator
+- Improved implementation of the layout.
```
Comment (altmannyarden) [inline Packs/CommonTypes/ReleaseNotes/3_2_42.md:4]: whats "implementation" mean?

Comment (Ni-Knight) [reply Packs/CommonTypes/ReleaseNotes/3_2_42.md:4]: its a fixed prefix to our notes... its stupid but its the only one that made any sense...
```
'Added support for '
'Added the '
'Fixed an issue '
'Improved implementation '
'Updated the Docker image to '
'You can now '
'Deprecated. '
'Deprecated the'
```

Comment (idovandijk) [reply Packs/CommonTypes/ReleaseNotes/3_2_42.md:4]: Maybe we should ask to add prefixes, looks like nobody took into account that we work on layouts, dashboards and playbooks which all have visual meanings in some way.

Comment (idovandijk) [reply Packs/CommonTypes/ReleaseNotes/3_2_42.md:4]: can you open a dev task for what is missing please? Shahaf said they would add it

Comment (Ni-Knight) [reply Packs/CommonTypes/ReleaseNotes/3_2_42.md:4]: @idovandijk - https://jira-hq.paloaltonetworks.local/browse/CIAC-3919

**Pull request closed without merging**